Applications/Viqui/vqCore.cxx

```diff
@@ -3387,6 +3393,37 @@ void vqCore::generateReport(QString path, bool generateVideo)
   this->exportKml(path);
 }
 
+//-----------------------------------------------------------------------------
+void vqCore::exportSvm(QString path)
+{
+  QFile file(path);
+  if (!file.open(QIODevice::WriteOnly |
+                 QIODevice::Truncate))
+    {
+    qDebug() << file.errorString();
+    QMessageBox::warning(0, QString(), "Unable to write SVM file.");
+    return;
+    }
+
+  if (this->SavedQueryPlan.isSimilarityQuery())
+    {
+    auto& query = *this->SavedQueryPlan.constSimilarityQuery();
```

Comment:
Pedantic: I know it's actually redundant, but `const auto` here is nice to show that the object won't be modified.